.sass-lint.yml

```diff
@@ -19,7 +19,7 @@ rules:
   # Disallows
   no-color-keywords: 1
   no-color-literals: 1
-  no-css-comments: 1
+  no-css-comments: 0
```

Comment:
It's to be able to put comments in the non-minified skeleton; they are removed in the minified version used in production.